contracts/facets/EmbalmerFacet.sol

```diff
@@ -120,6 +114,17 @@ contract EmbalmerFacet {
                     hashedShard: archaeologists[i].hashedShard
                 });
 
+            // Confirm that the archaeologist list is unique. This is done by
+            // checking that the archaeologist does not already exist from
+            // previous iterations in this loop.
+            if (
+                archaeologistExists(identifier, archaeologists[i].archAddress)
+            ) {
+                revert LibErrors.ArchaeologistListNotUnique(
+                    archaeologistAddresses
+                );
+            }
+
```

Comment:
(minor) -- this could possibly be the first logic run in this loop (before state is set)

Reply:
Moved